supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -198,8 +205,25 @@ trait SharedFileSystem extends PathFactory {
       PairOfFiles(src, dest)
     }
 
+    // A possibly staged version of the input file suitable for downstream processing, or just the original input
+    // file if no staging was required.
+    val staged: WomFile = value.mapFile { input =>
+      pathBuilders.collectFirst({ case http: HttpPathBuilder if HttpPathBuilder.accepts(input) => http }) match {
```

Comment:
In fact, this doesn't seem particularly specific to `Http` - if *any* path builder is willing to localize this path for us, shouldn't we let it?

Reply:
https://github.com/broadinstitute/cromwell/blob/develop/core/src/main/scala/cromwell/core/path/PathBuilderFactory.scala#L15